postgres-upgrade/Dockerfile

```diff
@@ -15,6 +15,7 @@ COPY ./repos/partner-management-services/db_upgrade_scripts .
 COPY ./repos/admin-services/db_upgrade_scripts .
 COPY ./repos/keymanager/db_upgrade_scripts .
 COPY ./repos/audit-manager/db_upgrade_scripts .
+COPY ./repos/resident-services/db_upgrade_scripts .
```

Comment:
hotlist is missing ?

Reply:
hotlist is part of ./repos/admin-services/db_upgrade_scripts . 
so it is not added.